src/SfxWeb/src/app/views/partition/backups/backups.component.html

```diff
@@ -27,7 +30,17 @@ <h4>Partition Restore Progress</h4>
     </div>
     
     <div *ngIf="partition && partition.replicas.isInitialized" class="detail-pane essen-pane">
-        <app-detail-list [list]="partition.partitionBackupInfo.partitionBackupList.collection" [listSettings]="partitionBackupListSettings" ></app-detail-list>
+        <app-detail-list [list]="backupList" [listSettings]="partitionBackupListSettings" >
+            <app-dual-date-picker [minDate]="minDate" [maxDate]="maxDate"
+            [currentStartDate]="startDate" [currentEndDate]="endDate" (dateChanged)="setNewDates($event)">
+            <div startTime>
+                <input type="time" [(ngModel)]="startTime" placeholder="HH:mm:ss" class="input-flat clear-disabled" ng-model-options="{ timezone: 'UTC' }" (change)="startTimeChange()" />
+            </div>
+            <div toTime>
+                <input type="time" [(ngModel)]="endTime" placeholder="HH:mm:ss" class="input-flat clear-disabled" ng-model-options="{ timezone: 'UTC' }" (change)="endTimeChange()" />
```

Comment:
is it necessary to use another variable to track the h/m/s part of the time? would using startDate/EndDate for these pickers have any issues? Shouldnt they just update that part of the time?

Reply:
input value goes to startTime/endTime as a string. if I use startDate/endDate in ngModel, it overrides the entire date with the time string.